group_project_v2/project_api/api_implementation.py

```diff
@@ -19,39 +14,18 @@
 SUBMISSION_API = '/'.join([API_PREFIX, 'submissions'])
 GROUP_API = '/'.join([API_PREFIX, 'groups'])
 COURSES_API = '/'.join([API_PREFIX, 'courses'])
+PROJECTS_API = '/'.join([API_PREFIX, 'projects'])
 
 
-DEFAULT_EXPIRATION_TIME = timedelta(seconds=10)
-
-
-# pylint:disable=too-many-instance-attributes
-class UserDetails(object):
-    def __init__(self, **kwargs):
-        self.id = kwargs.get('id', None)
-        self.email = kwargs.get('email', None)
-        self.username = kwargs.get('username', None)
-        self.uri = kwargs.get('uri', None)
-        self.first_name = kwargs.get('first_name', None)
-        self.last_name = kwargs.get('last_name', None)
-        self._full_name = kwargs.get('full_name', None)
-        self.gender = kwargs.get('gender', None)
-        self.avatar_url = kwargs.get('avatar_url', None)
-        self.city = kwargs.get('city', None)
-        self.country = kwargs.get('country', None)
-        self.is_active = kwargs.get('is_active', None)
-        self.level_of_education = kwargs.get('level_of_education', None)
-        self.organization = kwargs.get('organization', None)
+def do_not_cache(reason):  # pylint:disable=unused-argument
```

Comment:
@bradenmacdonald does not do anything now and there are no plans so far to change "caching" mechanism so that it would be required.